x-pack/plugins/apm/server/routes/suggestions/get_suggestions.ts

```diff
@@ -37,6 +42,10 @@ export async function getSuggestions({
       field,
       size,
       string,
+      ...(start &&
+        end && {
+          index_filter: rangeQuery(start, end)[0],
```

Comment:
This doesn't read super well. Not sure if we should just not use the `rangeQuery` helper

Reply:
https://github.com/elastic/kibana/pull/126679/commits/760f49a445560b6c8052a257a1a555ea67a86a88